PGM/src/main/java/tc/oc/pgm/polls/Poll.java

```diff
@@ -1,31 +1,32 @@
 package tc.oc.pgm.polls;
 
+import com.google.inject.Inject;
+import net.md_5.bungee.api.ChatColor;
+import tc.oc.commons.core.chat.Audiences;
+
 import java.util.HashMap;
 import java.util.Map;
 
-import org.bukkit.ChatColor;
-import org.bukkit.Server;
-
 public abstract class Poll implements Runnable {
-    protected final Map<String, Boolean> votes = new HashMap<String, Boolean>();
-    protected final long startTime = System.currentTimeMillis();
-    protected final PollManager pollManager;
-    protected final Server server;
-    protected String initiator;
-    protected int timeLeftSeconds;
 
     public static String boldAqua = ChatColor.BOLD + "" + ChatColor.AQUA;
     public static String normalize = ChatColor.RESET + "" + ChatColor.DARK_AQUA;
-    public static String seperator = ChatColor.RESET + " | ";
+    public static String separator = ChatColor.RESET + " | ";
 
-    public Poll(PollManager pollManager, Server server, String initiator) {
+    protected final PollManager pollManager;
+    protected final String initiator;
```

Comment:
I would suggest changing the `initiator` to be a `PlayerId` so that you can fix the bug where nicked players are exposed if they create a poll.

Reply:
that feels like it would be a bit restrictive, if other systems want to use polls (say the veto system for tournaments) forcing it into a player id may not be that good

Reply:
Tournaments are an entirely different system. Besides, if you were to refactor this for teams, it would just make more complicated. It's a reasonable assumption that a poll is initiated by a person and not something else.